solve.py

```diff
@@ -0,0 +1,51 @@
+import os.path
+
+def grab_input(rosalind_problem_name, downloads_directory):
+    """
+    Opens a text file representing rosalind input to solve a problem.
+    Pass in the name of the current rosalind problem name to look it up
+    in your downloads directory, where rosalind text files typically end up.
+
+    >>> grab_input('rosalind_ini3.txt')
+    wpoR2sT77ULTxwXaC9DNJKsM2jSUFlaDvhySPNATyhO92brubeDVnRXnx6YYzS88UHolodact
+    ylusbDN4pAeMJiGTF0IWa0u8VSWrBcJXmog5JI0kDVyklRxUxkCZihLQ1aJwqFhemilasiusw
+    hlfQ1AxRYA6H3QNFTcNnNiWfn7cuE.
+    65 76 135 144
+    """
+    if downloads_directory is None:
+        downloads_directory = '~/Downloads'
+    downloads_directory = os.path.expanduser(downloads_directory)
+    return open(os.path.join(downloads_directory, rosalind_problem_name), 'r').read()
+
+if __name__ == '__main__':
+    # you've called this from the command line
+    import argparse
+    import importlib
+
+    parser = argparse.ArgumentParser(
+        description=(
+            '''
+            Solve a rosalind problem. Make sure you name your python file that contains
+            the `main` function the same thing as the text file you downloaded from rosalind.
+            '''
+        )
+    )
+
+    parser.add_argument(
+        'problem_name',
+        action='store',
+        help='the name of the file you downloaded for this problem, minus the extension'
+    )
+
+    parser.add_argument(
+        '--downloads-directory',
+        dest='downloads_directory',
```

Comment:
So `dest=` chooses the variable in which the input is stored, right?

Based on what I read, in `+$> python solve.py rosalind_ini3 --downloads_directory=~/Desktop`,  the default `dest` would be `downloads_directory` anyway, since "`ArgumentParser` generates the value of `dest` by taking the first long option string and stripping away the initial `--` string"

Could we get away with not setting a `dest`?